nuntium/templatetags/nuntium_tags.py

```diff
@@ -35,3 +38,15 @@ def join_with_commas(obj_list):
 @register.assignment_tag(takes_context=True)
 def assignment_url_with_subdomain(context, view, subdomain=UNSET, *args, **kwargs):
     return subdomainsurls(context, view, subdomain, *args, **kwargs)
+
+
+@register.simple_tag(takes_context=True)
+def custom_stylesheet(context):
+    output = ''
+    if hasattr(context['request'], 'subdomain') and context['request'].subdomain is not None:
+        path = os.path.join(context['request'].subdomain, 'css/custom.css')
```

Comment:
My reasoning for this was that if you want to add quite a bit of custom static content - e.g images etc, being able to group it all in one place would make things easier to manage.